.evergreen/print-compass-env.sh

```diff
@@ -43,8 +43,8 @@ if [[ "${EVERGREEN_PROJECT}" == "10gen-compass-main" ]]; then
         month=$(escapeLeadingZero "${ts[1]}")
         day=$(escapeLeadingZero "${ts[2]}")
         hour=$(escapeLeadingZero "${ts[3]}")
-        minute="${ts[4]}"
-        second="${ts[5]}"
+        minute=$(escapeLeadingZero "${ts[4]}")
+        second=$(escapeLeadingZero "${ts[5]}")
```

Comment:
Small drive-by because CI failed for this PR in [this run](https://spruce.mongodb.com/version/684a1a0e27c4e80007faaa74/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

When `hour is 00 and minutes < 10`, the version is as `yymmdd-dev.0..`, which is invalid